etcdserver/server.go

```diff
@@ -374,6 +377,8 @@ func NewServer(cfg *ServerConfig) (*EtcdServer, error) {
 			srv.compactor = compactor.NewPeriodic(h, srv.kv, srv)
 			srv.compactor.Run()
 		}
+
+		srv.authStore = auth.NewAuthStore(srv.be)
```

Comment:
probably we should do something like lessor since auth will also keep some in memory state?

check https://github.com/coreos/etcd/blob/master/lease/lessor.go#L312